src/Task/TestFramework/TestRunner.php

```diff
@@ -127,7 +127,7 @@ public function run(): void {
       $this->startServers();
     }
 
-    if ($this->sut) {
+    if ($this->sut && $this->sut->getPackageName() != 'acquia/drupal-recommended-project') {
```

Comment:
ORCA interprets the Drupal core tests as the _template's_ tests, which is bad because core tests take hours to run.